lib/verifyWrapper.js

```diff
@@ -0,0 +1,29 @@
+var jwt = require ('./jwt');
+
+function verifyWrapper (verify, strategyOptions, authParams) {
```

Comment:
Lets get some comments in here as well ... I'd hate to have to try and figure out this logic later 😆